### ISSUE TITLE
Exception from one parent

### ISSUE THREAD

**Issue by shizacat** [amenezes/aiopytesseract#4]
### Need/Ideas Statement

Now check exception very hard. Need always take all exception.
May create from one, or two. But it is better from one.

Example:

```python
class TesseractRuntimeError(RuntimeError):
    pass


class TesseractError(Exception):
    """Base exception for tesseract"""

    def __init__(self, message=""):
        self.message = message
    
    def __str__(self):
        return self.message


class PSMInvalidException(TesseractError):    
    def __init__(self, message="PSM Invalid"):
        super().__init__(message)


class OEMInvalidException(TesseractError):
    def __init__(self, message="OEM Invalid"):
        super().__init__(message)


class NoSuchFileException(TesseractError):
    def __init__(self, message="No such file"):
        super().__init__(message)


class LanguageInvalidException(TesseractError):
    def __init__(self, message="Language invalid"):
        super().__init__(message)

```

It would be great if and 'TesseractRuntimeError'  was also inherited from 'TesseractError'. On you decision...

**Comment by amenezes** [comment]
Yes, you absolutely right.

I really made a mess with the exceptions :sweat_smile: 

Could you provide a PR with this corrections?

A few more for the list:
- https://github.com/amenezes/aiopytesseract/blob/master/aiopytesseract/commands.py#L94
- https://github.com/amenezes/aiopytesseract/blob/master/aiopytesseract/commands.py#L129
- https://github.com/amenezes/aiopytesseract/blob/master/aiopytesseract/commands.py#L433

Maybe this examples can be converted to `TesseractTimeoutError` or something like that.

**Comment by amenezes** [comment]
fixed by PR #5
